src/core/Class.js

```diff
@@ -90,6 +76,20 @@ export class Class {
 		return this;
 	}
 
+	_initHooksCalled = false;

```

Comment:
Since this field is set later on in `callInitHooks()` I have defined it here in advance.